packages/bind/spec/bindingHandlerBehaviors.ts

```diff
@@ -53,6 +53,11 @@ describe('BindingHandler behaviors', function () {
       var xCalls = 0,
         yCalls = 0
       bindingHandlers.fnHandler = class extends BindingHandler {
+
+        v : any
```

Comment:
just looking at just this file, it's unclear what v, x, y, computed are doing here

Reply:
Some tests are very interesting.. The member variables are now declared.